### ISSUE TITLE
"immediate" renderer looks broken

### ISSUE THREAD

**Issue by abingham** [sixty-north/gjenta#29]
```
$ gjenta quickplay --renderer=immediate input.gj                                                                                                                                                                                                                         
Fragment(text=">>> ", kind=Kind.Prompt, start_at=None, finish_at=None)Fragment(text="x = input()", kind=Kind.Input, start_at=None, finish_at=None)Fragment(text="
", kind=Kind.Input, start_at=None, finish_at=None)Fragment(text="foobar", kind=Kind.Input, start_at=None, finish_at=None)Fragment(text="
", kind=Kind.Input, start_at=None, finish_at=None)Fragment(text="", kind=Kind.Output, start_at=None, finish_at=None)Fragment(text=">>> ", kind=Kind.Prompt, start_at=None, finish_at=None)Fragment(text="print(x)", kind=Kind.Input, start_at=None, finish_at=None)Fragment(text="
", kind=Kind.Input, start_at=None, finish_at=None)Fragment(text="foobar
", kind=Kind.Output, start_at=None, finish_at=None)
```

**Comment by rob-smallshire** [comment]
I want to take some advice about dividends as an alternative.

**Comment by abingham** [comment]
Fixed in 600c28d2f3f6810c59094dcdf193a90f89f1a8e5.
